### DISCUSSION TITLE
This NodeIterator is no longer working.

### DISCUSSION THREAD

**Discussion by faab007nl** [requarks/wiki#5360] [Error / Bug Report]
I'm getting the following error when trying to save a page. 

Error when running job render-page: This NodeIterator is no longer working. More than undefined iterators are being used concurrently. You can increase the 'concurrentNodeIterators' option to make this error go away.

How can i change this option?

**Comment by Itimol82** [comment]
Translate to Arabic

**Reply by faab007nl** [reply]
What do you mean?

**Reply by Greazi-Times** [reply]
I translated the whole text he send into Arabic specially for you. 😁
Took me only 10 sec
أتلقى الخطأ التالي عند محاولة حفظ صفحة.

خطأ عند تشغيل صفحة عرض المهمة: هذا NodeIterator لم يعد يعمل. يتم استخدام أكثر من مكررات غير محددة بشكل متزامن. يمكنك زيادة خيار "concurrentNodeIterators" لإزالة هذا الخطأ.

كيف يمكنني تغيير هذا الخيار؟

**Comment by Greazi-Times** [comment]
> I'm getting the following error when trying to save a page.
> 
> Error when running job render-page: This NodeIterator is no longer working. More than undefined iterators are being used concurrently. You can increase the 'concurrentNodeIterators' option to make this error go away.
> 
> How can i change this option?

This only happens when you have more than 4 tabs.
